snap/snapcraft.yaml

```diff
@@ -0,0 +1,34 @@
+name: nsntrace
+adopt-info: nsntrace
+base: core20
+summary: Namespaced network tracer
+description: |
+  Perform network trace of a single process by using network namespaces.
+
+grade: stable
+confinement: classic
+
+parts:
+  nsntrace:
+    plugin: autotools
+    source-type: git
+    override-pull: | # This override tells snapcraft to always build and version using latest tag
+      snapcraftctl pull
+      last_tag="$(git describe --tags --abbrev=0)"
```

Comment:
Just a drive-by comment (not even knowing what nsntrace is :D):
You might want to add a `--match 'v*'` here to avoid using potential tags that aren't releases.

Something like this:
```
last_tag="$(git describe --tags --abbrev=0 --match 'v*')"
```

Reply:
Sounds lika a good idea! Thanks for driving by.